monai/transforms/utility/array.py

```diff
@@ -127,20 +127,22 @@ class AsChannelFirst(Transform):
 
     Args:
         channel_dim: which dimension of input image is the channel, default is the last dimension.
+            if channel_dim is a sequence, the transform will move the channel dimensions to the front.
     """
 
     backend = [TransformBackends.TORCH, TransformBackends.NUMPY]
 
-    def __init__(self, channel_dim: int = -1) -> None:
-        if not (isinstance(channel_dim, int) and channel_dim >= -1):
-            raise AssertionError("invalid channel dimension.")
-        self.channel_dim = channel_dim
+    def __init__(self, channel_dim: Union[int, Sequence[int]] = -1) -> None:
```

Comment:
this transform is deprecated, if we still want this feature, it should be in `EnsureChannelFirst` cc @ericspod

Reply:
This was related to another task you were working on. If we haven't had the need to implement this I would close this PR and return the concept later, though it seems that `Transpose` has similar behaviour anyhow.